sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -251,7 +251,7 @@ object FunctionRegistry {
     expression[Log10]("log10"),
     expression[Log1p]("log1p"),
     expression[Log2]("log2"),
-    expression[Log]("ln"),
+    expression[Log]("ln", true),
```

Comment:
Why don't we directly use `ln` in `Log`'s `prettyName`?

Reply:
I reference https://github.com/apache/spark/blob/a89006aba03a623960e5c4c6864ca8c899c81db9/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/mathExpressions.scala#L66.
So the default pretty name of `Log` is 'LOG',  I think we can use this method to make it as 'ln'.

Reply:
We should just directly override when it's possible because the tags are lost when it copies. That's defined in the parent node like that just to cover other general cases.

Reply:
OK.